client/src/index.js

```diff
@@ -1,11 +1,12 @@
 import React from 'react'
 import ReactDOM from 'react-dom/client'
-import App from './App'
```

Comment:
don't forget to add back App component, once code is finished

Reply:
index js shouldn't be touched, app is not working please make everything work, and don't use underscore for components, use CamelCase.